src/main/java/net/sf/jabref/importer/fileformat/MedlineImporter.java

```diff
@@ -61,7 +65,7 @@ public String getId() {
 
     @Override
     public String getDescription() {
-        return null;
+        return "Importer for the Medline format.";
```

Comment:
Have a look at the link for the MedlinePlainImporter, where the elements are described, then you will also find a link fpr the medline xml format. This link could be added here.